src/main/java/com/shaft/tools/io/internal/AllureManager.java

```diff
@@ -53,9 +53,8 @@ private static void copyAndOpenAllure() {
     }
 
     private static String renameAllureReport() {
```

Comment:
@MohabMohie i need your opinion here , i did change to make any allure report named with generation date whether accumulate flag is enabled or not .